Cython/Debugger/Tests/test_libcython_in_gdb.py

```diff
@@ -60,7 +60,7 @@ class DebugTestCase(unittest.TestCase):
     """
 
     def __init__(self, name):
-        super().__init__(name)
+        super(DebugTestCase, self).__init__(name)
```

Comment:
Why was this needed?

Reply:
The class [uses](https://github.com/cython/cython/blob/b31623d5c049063849c7fd4cdf5e72be2ddcf4a7/Cython/Debugger/Tests/test_libcython_in_gdb.py#L55) a [six](https://six.readthedocs.io) based [method](https://github.com/cython/cython/blob/0e1026cb7c767450a88ab822e8eda2b451dee3c6/Cython/Utils.py#L587) to add a metaclass in a Python 2 and 3 compatible way. This results in the class inheriting only from the metaclass that created it. [Here is a StackOverflow question on the topic](https://stackoverflow.com/q/58475561/3476782).

Reply:
Ok, then let's get rid of six for this instead. The master branch is Py3-only.